docs/pages/core_lib_doc/cache.md

```diff
@@ -27,8 +27,17 @@ def get_foo(foo_id):
 def set_foo(self, foo_id, foo_value):
     ... # update the value
 ```
+#### Code Explained:
+- <b>CACHE_KEY_FOO:</b> This variable holds a string representing the cache key format. It contains a placeholder {foo_id} which will be replaced with the actual foo_id value.
```

Comment:
Whatever you have, it's in the directory. like cache and job should have its on menu item in the root level.